config/deploy/production.rb

```diff
@@ -14,6 +14,7 @@
 server "lib-proc6", user: "deploy", roles: %w[worker]
 server "lib-proc7", user: "deploy", roles: %w[worker]
 server "lib-proc8", user: "deploy", roles: %w[worker]
+server "lib-proc9", user: "deploy", roles: %w[worker]
```

Comment:
We should remove this config; the box is not yet provisioned.

Reply:
👍